vip-jetpack/vip-jetpack.php

```diff
@@ -91,6 +91,24 @@
 	return $value;
 }, 9999 );
 
+/**
+ * Allow incremental syncing via cron to take longer than the default 30 seconds
```

Comment:
```suggestion
 * Allow incremental syncing via cron to take longer than the default 30 seconds.
 *
 * This will allow more items to be processed per cron event, while leaving a small buffer between completion and the start of the next event (the event interval is 5 mins).
```